### ISSUE TITLE
[Worldguard] issues with zones

### ISSUE THREAD

**Issue by charlieTheBotDev** [keepcalm/BukkitForge#147]
I made a zone to protect my spawn and added flags to it to stop creeper damage, and mob spawning as well as TNT explosions.

The mob spawn prevention works perfectly

If a creeper follows you into the zone, it still explodes and damages the environment.

I'm not sure if it's down to me using the wrong version of worldguard though, I have:

Worldedit 5.4.5 (modified for bukkitforge)
Worldguard 5.6.5

**Comment by keepcalm** [comment]
No, it's probably me not passing events properly. Unfortunately events take time to add, but they're getting somewhere.

**Comment by charlieTheBotDev** [comment]
That's cool. I'll test the other flags for worldguard and see what happens.

**Comment by Chasgasm** [comment]
Yeah I've noticed WorldGuard won't block creeper block damage, tnt block damage, fire spread, lava flow, etc as well even when set to true in the config. Glad to know you are working so hard on this keepcalm.

**Comment by delsolsi213** [comment]
you can do /gamerule mobgriefing false i believe its a vanilla command and it stops creepers explosions and with essentials i  blocked tnt explosion fire spreading etc

**Comment by shakey2** [comment]
Actually MCprotect funnily enough has the exact same problems with mobs following you into protections and being able to attack, once they lose target though they stop aggro-ing inside the protection.

**Comment by ghost** [comment]
Guys just use /gamerule mobGriefing false (its built in vanilla) and creepers ghasts and enderman cant destroy/pickup blocks :) oh yeah and withers

**Comment by charlieTheBotDev** [comment]
I only want it inside regions (i.e. spawn and main cities) and don't want to remove it globally so using that gamerule isn't a good solution for me

**Comment by shakey2** [comment]
I know about gamerule but I actually just want logblock to log mod blocks properly than anything. Protection is nice but its better to catch them because there is always gonna be a way to bypass protection on a modded server like mine. After that it'd be nice to get the essentials permission nodes for controlling block placement/destruction/interaction and item use everything after that is just bonuses to me.

**Comment by Mastergalen** [comment]
Build 110 seems to fix player move events. Greeting and farewell flags now work perfectly.

TNT and creeper-explosion flag still do not work though.

**Comment by charlieTheBotDev** [comment]
Well that's good to know, at least I don't have to test the TNT/Creeper flags again, thanks for the info Mastergalen

Edit: After playing a little while with /gamerule mobGriefing false I've found I quite like the lack of destruction globally so I'll probably just stay as I am. The issue obviously does still need to be fixed though ;)
